unsubscribe-automation/src/server.ts

```diff
@@ -0,0 +1,54 @@
+import fastify from "fastify";
+import cors from "@fastify/cors";
+import { autoUnsubscribe } from "./main";
+import { z } from "zod";
+
+const server = fastify();
+
+// Register CORS
+server.register(cors, {
+  origin: process.env.CORS_ORIGIN || "http://localhost:3000",
+  methods: ["GET", "POST"],
+});
+
+const UnsubscribeSchema = z.object({
```

Comment:
Start schemas with a lowercase letter. ie `UnsubscribeSchema`.
This is minor, but just keeps it consistent with the rest of the project. (I've seen other projects use the style you have here, but we don't)
Also zod docs, start with lowercase: https://zod.dev/?id=basic-usage

Reply:
Done